Assets/HoloToolkit/Input/Scripts/InputEvents/SourceClickedEventData.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    /// <summary>
+    /// Describes an input event that involves content manipulation.
```

Comment:
I'd recommend editing this comment. This wrapper is mainly for air tap correct?
Avoid using the word "manipulation". It could be confusing to some.